pkg/repository/prisma/dbsqlc/workflows.sql

```diff
@@ -82,7 +82,8 @@ SELECT
 FROM
     "Workflow" as workflows
 WHERE
-    workflows."tenantId" = $1 AND
+    workflows."tenantId" = @tenantId::uuid AND
+    workflows."name" LIKE '%' || @name::text || '%' AND
```

Comment:
```suggestion
    ( @name::text  IS NULL OR workflows."name" LIKE '%' || @name::text || '%' AND)
```
We should null check the input first,
we might need to use sqlc.nargs here

Reply:
```
AND
        (
            sqlc.narg('search')::text IS NULL OR
            workflow.name like concat('%', sqlc.narg('search')::text, '%')
        )
```

Also, curious how SQLC interprets the `||` is it the same as concat?

Reply:
|| is just string concatenation  - standardized it to your way